bin/plugin/commands/common.js

```diff
@@ -68,5 +81,52 @@ exports.getModuleData = async ( modulesDir ) => {
 
 			return moduleData;
 		} )
-		.filter( ( moduleData ) => moduleData.name && moduleData.description );
+		.filter( ( moduleData ) => {
+			const moduleDetails = JSON.stringify( moduleData );
+			if ( ! moduleData.name ) {
+				log(
+					formats.warning(
+						`This module was not included due is missing a required property 'name', Details: ${ moduleDetails }`
+					)
+				);
+			}
+
+			if ( ! moduleData.description ) {
+				log(
+					formats.warning(
+						`This module was not included due is missing a required property 'description', Details: ${ moduleDetails }`
+					)
+				);
+			}
+
+			if ( typeof moduleData.experimental === 'undefined' ) {
+				log(
+					formats.warning(
+						`This module was not included due is missing a required property 'experimental', Details: ${ moduleDetails }`
+					)
+				);
+			}
```

Comment:
Please, return `false` early to avoid having multiple warning messages for the same module if it has multiple properties empty. Or maybe would be better to determine which properties are missed and then output a single unified warning message with all missing properties. Something like this:

```
This module was not included because it misses required properties: 'name', 'experimental'.
```